2021-07-13 01:41:41,082:	DEBUG:	RulePullHandler.py:PullAndSave():92	have no data from cloud
